onnxruntime/test/python/onnxruntime_test_python_cudagraph.py

```diff
@@ -84,8 +84,9 @@ def test_select_ep_to_run_cuda_graph(self):
         elif "CUDAExecutionProvider" in onnxrt.get_available_providers():
             providers = [("CUDAExecutionProvider", {"enable_cuda_graph": True})]
             self.run_model_with_cuda_graph(providers)
+            self.run_model_with_cuda_graph(providers, use_graph_annotation=True)
 
-    def run_model_with_cuda_graph(self, providers):
+    def run_model_with_cuda_graph(self, providers, use_graph_annotation=False):
```

Comment:
could we add a test with multiple annotations like:

capture with annotation 1
replay on input 1
capture with annotation 2
replay on input 2
capture with annotation 1
replay on input 1
capture with annotation 2
replay on input 2

where input 1 and input 2 have different input shapes.

Reply:
sure